src/main/java/io/swagger/api/impl/CaseApiServiceImpl.java

```diff
@@ -16,63 +16,69 @@
 @javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaJerseyServerCodegen", date = "2020-10-13T17:17:47.836Z")
```

Comment:
I realize the white spaces changed a bit in this file. My IDE was using 4 spaces for indentation and I realized afterwards that the file had been using tab indentation (with 4 spaces). Looking at other files the indentation isn't consistent though because some other files don't use tabs either. I didn't see a style coding convention for this repo but if there is one please let me know.